Assets/HoloToolkit/Utilities/Scripts/Editor/InputManagerAxesWindow.cs

```diff
@@ -0,0 +1,295 @@
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_STICK_CLICK", positiveButton = "joystick button 9", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "CONTROLLER_LEFT_MENU", positiveButton = "joystick button 6", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_MENU", positiveButton = "joystick button 7", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "CONTROLLER_LEFT_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 9 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 10 },
+            new InputManagerAxis() { name = "XBOX_SHARED_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 3 },
+            new InputManagerAxis() { name = "XBOX_LEFT_BUMPER", positiveButton = "joystick button 4", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_RIGHT_BUMPER", positiveButton = "joystick button 5", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_A", positiveButton = "joystick button 0", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_B", positiveButton = "joystick button 1", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_X", positiveButton = "joystick button 2", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_Y", positiveButton = "joystick button 3", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_DPAD_HORIZONTAL", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 6 },
+            new InputManagerAxis() { name = "XBOX_DPAD_VERTICAL", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 7 }
+        };
+
+        /// <summary>
+        /// Used to map AxisType from a useful name to the int value the InputManager wants.
+        /// </summary>
+        public enum AxisType
+        {
+            KeyOrMouseButton,
+            MouseMovement,
+            JoystickAxis
+        };
+
+        /// <summary>
+        /// Used to define an entire InputManagerAxis, with each variable defined by the same term the Inspector shows.
+        /// </summary>
+        public class InputManagerAxis
```

Comment:
This class could be private, unless it needs to be public, then it should be in its own file.